go/launcher/webdriver/webdriver.go

```diff
@@ -404,7 +416,7 @@ func processResponse(body io.Reader, value interface{}) (*jsonResp, error) {
 		return nil, errors.New(compName, fmt.Errorf("%v unmarshalling %q", err, respBody))
 	}
 
-	if errBody.Status == 0 && errBody.Error == "" {
+	if respBody.Status == nil || *respBody.Status == 0 {
 		return nil, errors.New(compName, fmt.Errorf("%v unmarshalling %q", jsonErr, string(bytes)))
```

Comment:
This should probably be `err` instead of `jsonErr`. Actually, it looks like `jsonErr` is never read in any meaningful situation -- the only time it's written to is immediately before returning (on line 405).